cli/command/image/build.go

```diff
@@ -170,6 +176,10 @@ func (out *lastProgressOutput) WriteProgress(prog progress.Progress) error {
 
 // nolint: gocyclo
 func runBuild(dockerCli command.Cli, options buildOptions) error {
+	if os.Getenv("DOCKER_BUILDKIT") != "" {
+		return runBuildBuildKit(dockerCli, options)
```

Comment:
Any reason not to use CLI flag?

Reply:
@AkihiroSuda because in the future I'd like to remove it.